src/pages/gen2/build-a-backend/auth/admin-actions/index.mdx

```diff
@@ -12,4 +12,59 @@ export function getStaticProps(context) {
 
 }
 
-We do not yet have out-of-the-box support for Admin queries in Amplify (Gen 2). You will have to a create a new [AWS Lambda](https://aws.amazon.com/lambda/) resource, and provide it with the necessary permissions to access the [Amazon Cognito Admin Actions](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_Operations.html) behind the scenes.
+<Callout warning>
+
+**Under active development:** The Authentication experience for Amplify (Gen 2) is under active development. The experience may change between versions of `@aws-amplify/backend`. Try it out and provide feedback at https://github.com/aws-amplify/amplify-backend/issues/new/choose
+
+</Callout>
+
+### Grant function access
+
+You can grant your backend function access to authentication resources, which will enable it to execute queries and operations against users and groups in your Cognito user pool.
+
+```ts title="amplify/auth/resource.ts"
+import { defineAuth, defineFunction } from '@aws-amplify/backend';
+
+const demoFunction = defineFunction({});
+
+export const auth = defineAuth({
+  name: 'myProjectFiles',
+  loginWith: { email: true },
+  access: (allow) => ([
+    allow.resource(demoFunction).to(['manageUsers']),
+  ])
+});
+```
+
+This will grant the backend function, `demoFunction` ability to query users and updates to user in the Cognito user pool. See below for detailed list of permissions it grants.
```

Comment:
```suggestion
This will grant `demoFunction` the ability to perform CRUD operations on users in the Cogntio UserPool created as part of `defineAuth`. See below for the detailed list of permissions.
```